code/modules/shuttle/emergency.dm

```diff
@@ -129,8 +129,9 @@
 		// shuttle timers use 1/10th seconds internally
 		SSshuttle.emergency.setTimer(ENGINES_START_TIME)
 		var/system_error = emagged ? "SYSTEM ERROR:" : null
+		var/emag_boost_message = pick("STRAP IN FOR EMERGENCY BURN", "PREPARE FOR LUDICROUS SPEED", "WHEN THIS BABY HITS 88 MILLION MILES PER HOUR YOU'RE GONNA SEE SOME SERIOUS SHIT", "MAKING THE JUMP TO %@ERROR^& IN UNDER 12 PARSECS", "INERTIAL DAMPENERS OFFLINE", "WHAT IF THE ENGINES MOVE THE UNIVERSE, NOT THE SHUTTLE?")
```

Comment:
Parsecs are a unit of distance, not time or speed. 0 / :100:

Reply:
😏

Reply:
I can't tell if you're joking or if you've never seen Star Wars.

Reply:
Star Wars is trash.